packages/synapse-react-client/src/components/Plot/Plot.tsx

```diff
@@ -0,0 +1,24 @@
+import { PlotParams } from 'react-plotly.js'
+import createPlotlyComponent from 'react-plotly.js/factory'
+import { lazy, Suspense } from 'react'
+
+// Lazy-load plotly.js since it is very large!
+const LazyLoadedPlot = lazy(async () => {
+  const Plotly = await import('plotly.js-basic-dist')
+  return { default: createPlotlyComponent(Plotly) }
+})
+
+/**
+ * Wrapper around react-plotly.js, lazy loads the plotly.js library.
+ * @param props
+ * @constructor
+ */
+export function Plot(props: PlotParams) {
+  return (
+    <Suspense>
+      <LazyLoadedPlot {...props} />
+    </Suspense>
+  )
+}
+
+export default Plot
```

Comment:
plotly.js-basic-dist is one of our largest dependencies -- so let's lazy load it so we don't use it unless we are rendering a Plot.

When using React.lazy, we must wrap in Suspense. This loader could be improved, but is fine for now.